src/actions/receive-profile.js

```diff
@@ -293,107 +289,10 @@ export function finalizeActiveTabProfileView(
   selectedThreadIndex: ThreadIndex,
   showTabOnly?: BrowsingContextID | null
 ): ThunkAction<void> {
-  return (dispatch, getState) => {
-    const hasUrlInfo = selectedThreadIndex !== null;
```

Comment:
Now we don't have to calculate any of those for active tab

Reply:
Yessss. I hate this code (I wrote it)